pages/_meta.json

```diff
@@ -2,15 +2,15 @@
   "index": "Introduction",
   "general": "General",
   "validators": "Validators",
-  "testnets": "Testnets",
+  "mainnet": "Mainnet",
   "integration_docs": "Integration documents",
   "developer_links": "Developer links",
   "governance": "Governance",
   "operational_checklist": "Operational checklist",
   "app": {
-    "title": "Testnet",
+    "title": "Mainnet",
     "type": "page",
-    "href": "https://v4.testnet.dydx.exchange",
+    "href": "https://v4-mainnet-docs.vercel.app/",
```

Comment:
Don't we want a custom domain for that? Relying on Vercel to this degree seems like a huge vendor lock-in.

Reply:
Certainly, when we have the domain ready, this should be updated. 
Actually, I'll put https://dydx.trade here, not to postpone.
Your commend, however :) is very good! This should link to the exchange FE, not to the documentation.

Will fix in PR#3